tests/integration-tests/src/main/graphql/com/apollographql/apollo3/integration/normalizer/schema.graphqls

```diff
@@ -88,10 +90,18 @@ interface Character {
 The `Date` scalar type represents date format.
 """
 scalar Date
+
+scalar Instant
+
 """
 UnsupportedType for testing
 """
 scalar UnsupportedType
+
+type InstantContainer {
+  instant: Instant
+}
```

Comment:
FWIW, I don't think the container is needed here, adding `Query.instant` would work as well for the purpose of the test